### DISCUSSION TITLE
The meta.json is not working after updating to "fumadocs-mdx" version 10

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [fuma-nama/fumadocs#839] [Q&A] [answered]
What am I missing here? 

<img width="1498" alt="image" src="https://github.com/user-attachments/assets/b9ecef0d-ab50-4a50-981d-038ce9124a0c">

![code](https://github.com/user-attachments/assets/db11fa4d-ae4b-4892-95e2-0237bbf0f445)

**Comment by fuma-nama** [comment, marked as answer]
You have to pass the dir options to `meta` too

**Reply by Ali-Hussein-dev** [reply]
I think this should be mentioned in the docs.

**Reply by fuma-nama** [reply]
Maybe we can add another section under https://fumadocs.vercel.app/docs/mdx/configuration#define-docs

**Reply by Ali-Hussein-dev** [reply]
Why not here https://fumadocs.vercel.app/docs/headless/page-conventions ?

**Reply by fuma-nama** [reply]
This is a source specific thing
